include/crocoddyl/multibody/actions/contact-fwddyn.hxx

```diff
@@ -163,6 +164,32 @@ DifferentialActionModelContactFwdDynamicsTpl<Scalar>::createData() {
   return boost::allocate_shared<Data>(Eigen::aligned_allocator<Data>(), this);
 }
 
+template <typename Scalar>
+void DifferentialActionModelContactFwdDynamicsTpl<Scalar>::quasiStatic(
```

Comment:
Did you create a unittest?
We should check it against the default implementation that I have coded quite time ago.